pom.xml

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.exist-db</groupId>
+        <artifactId>exist-apps-parent</artifactId>
+        <version>1.3</version>
+        <relativePath/>
+    </parent>
+
+    <groupId>org.exist-db</groupId>
+    <artifactId>atomic-wiki</artifactId>
+    <version>0.7-SNAPSHOT</version>
+
+    <name>Atomic Wiki</name>
```

Comment:
This should be `AtomicWiki` so the app properly updates from older versions instead of installing a parallel version

Reply:
Okay I can change that...